backend/src/extension.ts

```diff
@@ -129,7 +129,7 @@ export class YeomanUIPanel {
 			vscodeYouiEvents, 
 			outputChannel, 
 			this.logger, 
-			{genFilter: this.genFilter, messages: this.messages, defaultNpmPaths},
+			{genFilter: this.genFilter, messages: this.messages, data: _.get(uiOptions, "data"), defaultNpmPaths},
```

Comment:
data: _.get(uiOptions, "data", {})

Reply:
I do not want to create any object if it is not created by command. It is not our logic